workflow_tools.md

```diff
@@ -0,0 +1,32 @@
+## Purpose
+
+This page provides local policy and guidance for using workflow tools, such as CircleCI and GitHub Actions, on open-source repositories.
+
+This page covers the following topics:
+* [WorkFlow Tools Configuration](#workflow-tools)
+* [Secrets Management for Workflows](#workflow-secrets)
+
+
+## Workflow Tools
+
+Workflow tools such as CircleCI and Github Actions provide essential PR checks to effectively manage changes to code within repositories. The code for the repository workflow should be managed in the same way as all other code for the repository, requiring code owner review in order to commit any changes.
```

Comment:
This is not strictly true. The tools are able to do a lot more than just run on PRs. For example GH actions can create release artifacts for you / upload images to Docker hub etc. CircleCI has the ability for the same.

Reply:
That's a fair point; what I have written does not fully reflect how we currently use workflow tools. I'm not convinced we need any intro really, just added one for completeness. I've generalised the text, but happy to just remove the pre-amble if you'd rather.

Reply:
I do think it provides context here and also implies how we plan to use the tools, so I think it provides some benefit here, thanks for updating!